src/shapes/path.class.js

```diff
@@ -324,7 +384,39 @@
         width: deltaX,
         height: deltaY
       };
-    }
+    },
+
+    _setPositionDimensions: function (options) {
+      options || (options = {});
+      var calcDim = this._calcDimensions(options), origin,
+        sizeCorrection = options.correction || 0;
+      this.width = calcDim.width - sizeCorrection;
+      this.height = calcDim.height - sizeCorrection;
+      if (!options.fromSVG) {
+        origin = this.translateToGivenOrigin(
+          // this looks bad, but is one way to keep it optional for now.
+          new fabric.Point(
```

Comment:
I don't like this. Because it doesn't take into account the translation caused by stroke width

Reply:
resolved I believe